Example Apps/Example Swift/ProxyManager.swift

```diff
@@ -46,7 +46,8 @@ extension ProxyManager {
     /// - Parameter connectionType: The type of transport layer to use.
     func start(with proxyTransportType: ProxyTransportType) {
         delegate?.didChangeProxyState(ProxyState.searching)
-        sdlManager = SDLManager(configuration: proxyTransportType == .iap ? ProxyManager.connectIAP() : ProxyManager.connectTCP(), delegate: self)
+
+        sdlManager = SDLManager(configuration: (proxyTransportType == .iap) ? ProxyManager.iapConfiguration : ProxyManager.tcpConfiguration, delegate: self)
```

Comment:
```suggestion
        sdlManager = SDLManager(configuration: proxyTransportType == .iap ? ProxyManager.iapConfiguration : ProxyManager.tcpConfiguration, delegate: self)
```

Reply:
I purposefully added these parenthesis to clarify the order of operations here.